kde-apps/kmahjongg/kmahjongg-24.12.3.recipe

```diff
@@ -0,0 +1,158 @@
+	lib:libKMahjongg6$secondaryArchSuffix
+	# KF6
+	lib:libKF6Archive$secondaryArchSuffix
+	lib:libKF6Bookmarks$secondaryArchSuffix
+	lib:libKF6Codecs$secondaryArchSuffix
+	lib:libKF6ColorScheme$secondaryArchSuffix
+	lib:libKF6Completion$secondaryArchSuffix
+	lib:libKF6ConfigCore$secondaryArchSuffix
+	lib:libKF6ConfigGui$secondaryArchSuffix
+	lib:libKF6ConfigWidgets$secondaryArchSuffix
+	lib:libKF6CoreAddons$secondaryArchSuffix
+	lib:libKF6Crash$secondaryArchSuffix
+	lib:libKF6DBusAddons$secondaryArchSuffix
+	lib:libKF6GuiAddons$secondaryArchSuffix
+	lib:libKF6I18n$secondaryArchSuffix
+	lib:libKF6IconThemes$secondaryArchSuffix
+	lib:libKF6ItemViews$secondaryArchSuffix
+	lib:libKF6JobWidgets$secondaryArchSuffix
+	lib:libKF6KIOCore$secondaryArchSuffix
+	lib:libKF6NewStuffCore$secondaryArchSuffix
+	lib:libKF6NewStuffWidgets$secondaryArchSuffix
+	lib:libKF6Package$secondaryArchSuffix
+	lib:libKF6Service$secondaryArchSuffix
+	lib:libKF6Solid$secondaryArchSuffix
+	lib:libKF6TextEditor$secondaryArchSuffix
+	lib:libKF6TextWidgets$secondaryArchSuffix
+	lib:libKF6WidgetsAddons$secondaryArchSuffix
+	lib:libKF6WindowSystem$secondaryArchSuffix
+	lib:libKF6XmlGui$secondaryArchSuffix
+	# Qt6
+	lib:libQt6Core$secondaryArchSuffix
+	lib:libQt6DBus$secondaryArchSuffix
+	lib:libQt6Gui$secondaryArchSuffix
+	lib:libQt6Qml$secondaryArchSuffix
+	lib:libQt6Svg$secondaryArchSuffix
+	lib:libQt6SvgWidgets$secondaryArchSuffix
+	lib:libQt6Widgets$secondaryArchSuffix
+	lib:libQt6Xml$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libkdegames6$secondaryArchSuffix
+	devel:libKMahjongg6$secondaryArchSuffix
+	# KF6
+	devel:libKF6Archive$secondaryArchSuffix
+	devel:libKF6Bookmarks$secondaryArchSuffix
+	devel:libKF6Codecs$secondaryArchSuffix
+	devel:libKF6ColorScheme$secondaryArchSuffix
+	devel:libKF6Completion$secondaryArchSuffix
+	devel:libKF6ConfigCore$secondaryArchSuffix
+	devel:libKF6ConfigWidgets$secondaryArchSuffix
+	devel:libKF6CoreAddons$secondaryArchSuffix
+	devel:libKF6Crash$secondaryArchSuffix
+	devel:libKF6DBusAddons$secondaryArchSuffix
+	devel:libKF6DocTools$secondaryArchSuffix
+	devel:libKF6GuiAddons$secondaryArchSuffix
+	devel:libKF6I18n$secondaryArchSuffix
+	devel:libKF6IconThemes$secondaryArchSuffix
+	devel:libKF6ItemModels$secondaryArchSuffix
+	devel:libKF6ItemViews$secondaryArchSuffix
+	devel:libKF6JobWidgets$secondaryArchSuffix
+	devel:libKF6KIOCore$secondaryArchSuffix
+	devel:libKF6NewStuffCore$secondaryArchSuffix
+	devel:libKF6NewStuffWidgets$secondaryArchSuffix
+	devel:libKF6Package$secondaryArchSuffix
+	devel:libKF6Service$secondaryArchSuffix
+	devel:libKF6Solid$secondaryArchSuffix
+	devel:libKF6TextEditor$secondaryArchSuffix
+	devel:libKF6TextWidgets$secondaryArchSuffix
+	devel:libKF6WidgetsAddons$secondaryArchSuffix
+	devel:libKF6WindowSystem$secondaryArchSuffix
+	devel:libKF6XmlGui$secondaryArchSuffix
+        extra_cmake_modules$secondarextra_cmake_modules$secondaryArchSuffixyArchSuffix
```

Comment:
I guess something went wrong with copy/paste here ;) ECM should go at the top of the KF6 frameworks.